run-docker.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+docker run -it --rm --hostname dronecode-docker -v $(pwd):/home/dronecode/DronecodeSDK dronecode
```

Comment:
Do we need the hostname? And does that not conflict if you have multiple on the same machine, e.g. when testing with the different containers?

Reply:
its not necessary. leaving it out means a bash prompt of dronecode@<randomstringhere>. dronecode@dronecode-docker was a bit more intuitive.